addons/hr_calendar/static/src/calendar/filter_panel/calendar_filter_panel.js

```diff
@@ -0,0 +1,16 @@
+/** @odoo-module **/
+
+import { AttendeeCalendarFilterPanel } from "@calendar/views/attendee_calendar/filter_panel/attendee_calendar_filter_panel";
+import { patch } from "@web/core/utils/patch";
+
+
+patch(AttendeeCalendarFilterPanel.prototype, {
+    updateSelectCreateDialogProps(props) {
+        const updatedProps = super.updateSelectCreateDialogProps(props);
+        updatedProps.context = {
+            search_view_ref: 'hr_calendar.view_res_partner_filter_inherit_calendar',
+            search_default_type_team: 1,
+        };
+        return updatedProps;
+    }
+})
```

Comment:
:eyes:

Reply:
Hello, @arj-odoo 
In testing BE dala added point -> "My team" filter to select emp of current user's team, so he can easily added them to meeting In one click by selecting them (only when Employee app is installed). 

That's why this file exists. I'll also add a new line here.
Or were you asking for something else? 🤔

Reply:
No, I was only pointing the new line. I did not made a full review yet :-).

Reply:
Ahh, okay!